src/test/scala/cookbook/FSM.scala

```diff
@@ -7,16 +7,21 @@ import chisel3.util._
 
 /* ### How do I create a finite state machine?
  *
- * Use Chisel Enum to construct the states and switch & is to construct the FSM
+ * Use Chisel StrongEnum to construct the states and switch & is to construct the FSM
  * control logic
  */
+
 class DetectTwoOnes extends Module {
   val io = IO(new Bundle {
     val in = Input(Bool())
     val out = Output(Bool())
   })
 
-  val sNone :: sOne1 :: sTwo1s :: Nil = Enum(3)
+  object State extends ChiselEnum {
+    val sNone, sOne1, sTwo1s = Value
+  }
+  import State._
```

Comment:
Is this recommended practice? I'd lean towards having users be explicit about namespacing, eg using `State.sNone`

Reply:
I was gonna leave this completely up to users, and whether they want to keep their namespaces apart.

Reply:
Ultimately, it's up to the users, but we should set good examples in test code.